.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: required
 dist: xenial
 cache: bundler
 language: python
-python: "2.7"
+python: 3.7.1
```

Comment:
I'm not sure this is valid for this config key: https://docs.travis-ci.com/user/languages/python/#specifying-python-versions

Reply:
you mean it won't pick the .1 ? that seems right. I also see the paragraph below:

> If the specified version of Python is not available on the present build image, the job will attempt to download the suitable remote archive and make it available. 

But I think we are fine with 3.7.

Reply:
Xenial in Travis comes with 3.7.1 preinstalled: https://docs.travis-ci.com/user/reference/xenial/#python-support

And the build is using it

![Screenshot from 2020-04-22 18-04-10](https://user-images.githubusercontent.com/762088/80005400-ca8dc800-84c3-11ea-99b7-2d31fcec579e.png)